NetSpell.SpellChecker/Spelling.cs

```diff
@@ -940,13 +940,13 @@ public bool SpellCheck(string text, int startWordIndex)
         /// </param>
         /// <remarks>
         ///		This method sets the <see cref="Text"/> property to the word.
-        ///		Then calls <see cref="TestWord"/> on the word to generate the need
+        ///		Then calls <see cref="TestWord(string)"/> on the word to generate the need
```

Comment:
:bulb: Please leave a comment on dotnet/csharplang#320 in support of this future feature (which I've already implemented but needs approval from the language design team).

Reply:
This would be great actually. I documented an API recently and grew tired of listing out specific overloads. Will comment as much on the linked issue.